VMEncryption/main/handle.py

```diff
@@ -604,8 +604,8 @@ def mark_encryption(command, volume_type, disk_format_query):
 
 
 def is_daemon_running():
-    handler_path = os.path.join(os.getcwd(), __file__)
-    daemon_arg = "-daemon"
+    handler_path = os.path.join(os.getcwd(), __file__).encode('utf-8')
+    daemon_arg = b'-daemon'
```

Comment:
Will it be possible to add a unit test for this function so that we can run it against both py 2 and 3?

Reply:
That was bothering me too. I don't see an existing unit test wrapper for handle.py, and that blind spot may be how this slipped through prior testing.  I will look into moving this function out to a utility class, and make handle.py a little bit smaller   too.